pyproject.toml

```
@@ -20,12 +20,8 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.11"
-structlog = "^21.2.0"
-PyYAML = "^6.0.0"
-pydantic = "^1.9.0"
+python = ">=3.7,<3.12"
 typer = "^0.6.1"
```

Let's bump the min version to 0.7.0 for Python 3.11 support

```suggestion
typer = "^0.7.0"
```